e2e/evaluation_test.go

```diff
@@ -1,20 +1,29 @@
 package e2e_test
```

Comment:
This is a port from go test suite [1]

[1] -https://github.com/open-feature/go-sdk-contrib/blob/main/tests/flagd/pkg/integration/evaluation.go